### DISCUSSION TITLE
Harpoon2 Release

### DISCUSSION THREAD

**Discussion by ThePrimeagen** [ThePrimeagen/harpoon#550] [Announcements]
Harpoon2 is ready to release once the following is done:

* Bug Fixes 
* Documentation

...

sucks to suck.

**Comment by Faroukhamadi** [comment]
Skill issues preventing 2.0 release as always

**Comment by Lasuch69** [comment]
Can I integrate software renderer into harpoon's menu? I would like to have spinning donuts all over my Neovim config. :3

**Reply by ThePrimeagen** [reply]
Let me check the feature list...





























No

**Reply by Lasuch69** [reply]
Feels bad man :<

**Comment by ThePrimeagen** [comment]
todays open source work: 
* ui list did not allow for replace_at to be translated into a ui view.  in other words, the empty spaces would be stripped
* replace_at would cause duplicates.  the removal of duplicated items respects the "replace_at" style of a list.  this is a complicated feature :(
* bug fixes

trimming of the tickets work:
* #551 i think is likely just some weird spot in the history and isn't real issue
* #542 fixed

**Comment by surmish** [comment]
when will this be merged?

**Comment by romado77** [comment]
@ThePrimeagen, have you abandoned the project? Just curious if there are any plans for further development and maintenance of this project.

**Comment by raindev** [comment]
Having a co-maintainer(s) might be a solution for the continuity of the project.
